templates/guess_the_painter.html

```diff
@@ -10,15 +10,39 @@
              border="0"></center>
     <div>Answers:</div>
 
-    <form method="post">
+    <form method="post" id="painter_form_id">
         <ul class="entries">
             {% for painter in painters %}
-                <input id="chosen_painter_{{ painter.id }}" type="radio" name="chosen_painter"
-                       value="{{ painter.id }}">
-                <label for="chosen_painter_{{ painter.id }}">{{ painter.name }}</label>
+                <button id="chosen_painter_{{ painter.id }}" type="button" name="chosen_painter"
+                       value="{{ painter.id }}" onclick="setColor(this,'{{painter.id}}', '{{selected_painter.id}}')">
+               {{ painter.name }}
+                    </button>
                 <br/>
             {% endfor %}
         </ul>
-        <input type="submit" value="Submit"/>
     </form>
+<script>
+    function setColor(btn, correct, current) {
```

Comment:
well, the idea is right, but...
1) we should use jquery instead (it's a wrapper around js with many convenience methods and cross-browser compatibility)
2) transfer the js code in an external file, so that you don't use the SAME code in two diff places. please read this one: http://programmer.97things.oreilly.com/wiki/index.php/Don't_Repeat_Yourself